Assets/MixedRealityToolkit-Examples/Boundary/Scripts/BoundaryVisualizer.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using MixedRealityToolkit.Boundary;
+
+namespace MixedRealityToolkit.Examples.Boundary
+{
+    /// <summary>
+    /// Demo class to show different ways of using the boundary API
+    /// </summary>
+    public class BoundaryVisualizer : MonoBehaviour
+    {
+        [Tooltip("Material used to draw the rectangle bounds")]
+        public Material BoundsMaterial;
```

Comment:
private and serialized please.
if you need access from an outside class add a public property that uses the field as backing.